rpms/systemd/sources/FB--add-back-compat-libs.patch

```diff
@@ -79,8 +79,9 @@ index 0c27f81..40138b2 100644
 +	-Wl,--version-script=$(top_srcdir)/src/compat-libs/libsystemd-journal.sym
 +
 +libsystemd_journal_la_LIBADD = \
-+	$(libsystemd_internal_la_LIBADD) \
-+	$(libsystemd_journal_internal_la_LIBADD)
++	libsystemd-journal-internal.la \
```

Comment:
The patch removes the variables (they are just linker flags) and adds the respective .la archives. The quoted part then looks like this:
```
libsystemd_journal_la_LIBADD = \
	libsystemd-journal-internal.la \
	libsystemd-internal.la \
	libbasic.la
```
It may not be the most effective way, because the symbols from libbasic.la then end up in all those libraries where it was added, but the effectiveness wasn't important for me. I wanted to resolve the missing symbols issue, which I did with this patch.